### PR TITLE
Gumball play lab version 1

### PR DESCRIPTION
**Pull request by caleybrock**
Still need sounds

### CODE REVIEW

**Comment by caleybrock** [comment]
fyi @philbogle @cpirich 
This shouldn't be a risky change, since I'm just adding a new skin, but I wanted to let you know since it touches playlab.

File: apps/src/studio/skins.js [hunk 1]
```diff
@@ -16,6 +16,180 @@ var CLICK_VALUE = constants.CLICK_VALUE;
...
+    background: skin.assetUrl('background_wood.png'),
+  };
+
+  skin.avatarList = ["anais", "antony", "bananajoe", "darwin", "gumball", "nicole", "penny", "richard"];
+  skin.walkValues = [8, 8, 8, 12, 12, 8, 10, 12];
+
+  /**
+   * Sprite thumbs generated with:
+   * `brew install graphicsmagick`
+   * `gm convert +adjoin -crop 200x200 -resize 100x100 *spritesheet* output%02d.png`
```
Comment (breville) [inline apps/src/studio/skins.js:108]: true?

**Comment by breville** [comment]
lgtm

you might want to pngquant all the new images.

also @cpirich might be interested in the playlab extensions.

**Comment by caleybrock** [comment]
I'll do a follow up pngquant on these - I should do it for ice age too. I just want to get a few more assets in and do it all at once.

**Comment by cpirich** [comment]
Looks like you may have added some item strings back that were recently removed (e.g. itemRoo)

**Comment by caleybrock** [comment]
good catch @cpirich I think I accidentally added them back in during a merge. I'll remove in a followup pr.

**Pull request merged**